src/files/json.ts

```diff
@@ -22,9 +22,14 @@ async function readJSONText(file: BIDSFile): Promise<string> {
 
 export async function loadJSON(file: BIDSFile): Promise<Record<string, unknown>> {
   const text = await readJSONText(file) // Raise encoding errors
+  let parsedText;
   try {
-    return JSON.parse(text)
+    parsedText = JSON.parse(text)
   } catch (error) {
     throw { key: 'JSON_INVALID' } // Raise syntax errors
   }
+  if (Array.isArray(parsedText)) {
+    throw { key: 'JSON_NOT_AN_OBJECT' }
+  }
```

Comment:
It could also be a number or string. We want to check that it is actually an object, I think.